fetch/api/basic/scheme-about.any.js

```diff
@@ -1,34 +1,17 @@
 // META: script=../resources/utils.js
 
-function checkFetchResponse(url, method, desc) {
-  if (!desc) {
-    var cut = (url.length >= 40) ? "[...]" : "";
-    cut += " (" + method + ")"
-    desc = "Fetching " + url.substring(0, 40) + cut + " is OK"
-  }
+function checkKoUrl(url, method) {
```

Comment:
This isn't new, but having read the whole test and scratched my head I have no idea what "KO" means, probably "not OK" given the `promise_rejects` bit. Can you add a comment?